pkg/config/testdata/test_other_yamls/local-project.yaml

```diff
@@ -17,3 +17,4 @@ spec:
   - name: project-admins
     type: Group
     role: ProjectAdmin
+    dn: cn=Developers,ou=users,dc=example,dc=com
```

Comment:
as above, I fixed this bug by removing the whole `project-admins`

Reply:
will be fixed after master merge